export_options.go

```diff
@@ -446,6 +549,13 @@ func (g ExportOptionsGenerator) generateExportOptions(exportMethod exportoptions
 		}
 	}
 
-	exportOpts = addXcode9Properties(exportOpts, codeSignGroup.Certificate().TeamID, codeSignGroup.Certificate().CommonName, exportCodeSignStyle, exportProfileMapping, xcodeManaged)
+	if xcodeMajorVersion >= 9 {
```

Comment:
I just removed the if statement as Xcode can not be >= 9 at this point ([we return in case Xcode < 9 earlier](https://github.com/bitrise-steplib/steps-xcode-archive/pull/198/files#diff-e280eda0b16632926e9346f7af8c3037R446)).